elasticdl/python/elasticdl/layers/embedding.py

```diff
@@ -170,12 +176,23 @@ def call(self, input):
         ids = tf.convert_to_tensor(input, name="embedding_ids")
         flat_ids = tf.reshape(ids, [-1])
         unique_ids, idx = tf.unique(flat_ids)
+
+        # There is a memory leak using `tf.py_function` with eager execution.
+        # So, we directly call `lookup_embedding` when inputs are EagerTensor.
```

Comment:
There is a memory leak when using `tf.py_function` with eager execution.
So, we only use `tf.py_function` in graph mode.

The comment below can be deleted.